stories/icon-tab-bar/icon-tab-bar.stories.js

```diff
@@ -0,0 +1,1431 @@
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--critical">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab14">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+                <i class="sap-icon--cart" role="presentation"></i>
+          </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">10</span>
+        </div>
+      </a>
+    </li>
+    <li role="presentation" class="fd-icon-tab-bar__item fd-icon-tab-bar__item--informative">
+      <a role="tab" class="fd-icon-tab-bar__tab" href="#" id="tab15">
+        <div class="fd-icon-tab-bar__container">
+          <span class="fd-icon-tab-bar__icon">
+            <i class="sap-icon--cart" role="presentation"></i>
+          </span>
+          <span class="fd-icon-tab-bar__badge fd-icon-tab-bar__badge--animated"></span>
+          <span class="fd-icon-tab-bar__counter">10</span>
+        </div>
+      </a>
+    </li>
+  </ul>
+</div>
+`;
+badge.storyName = 'Badges';
+badge.parameters = {
+    docs: {
+        storyDescription: `The Tab attention badge is applied as a visual eye-catcher to indicate a change within a Tab. <br>
+        The badge is a span element with <code>fd-icon-tab-bar\\_\\_badge</code> class. It can be animated by adding an additional <code> fd-icon-tab-bar\\_\\_badge--animated</code> modifier class. In this case the badge appears for a few milliseconds and then disappears.`
+    }
+};
+
+export const respPaddings = () => `<div class="fd-icon-tab-bar fd-icon-tab-bar--responsive-paddings">
+    <ul role="tablist" class="fd-icon-tab-bar__header">  
+        <li role="presentation" class="fd-icon-tab-bar__item">
+            <a role="tab" class="fd-icon-tab-bar__tab" href="#section11" id="tab1">
+                <span class="fd-icon-tab-bar__tag">Section 1</span>
+                <span class="fd-icon-tab-bar__badge"></span>
+            </a>
+        </li>
+        <li role="presentation" class="fd-icon-tab-bar__item">
+            <a role="tab" class="fd-icon-tab-bar__tab" href="#section12" aria-selected="true" id="tab2">
+                <span class="fd-icon-tab-bar__tag">Section 2</span>
+            </a>
+        </li>
+        <li role="presentation" class="fd-icon-tab-bar__item">
+            <a role="tab" class="fd-icon-tab-bar__tab" href="#section13" id="tab3">
+                <span class="fd-icon-tab-bar__tag">Section 3</span>
+            </a>
+        </li>
+    </ul>
+</div>
+`;
+respPaddings.storyName = 'Responsive Paddings';
+respPaddings.parameters = {
+    docs: {
+        storyDescription: 'For an Icon Tab Bar with responsive paddings that adjust based on the screen size add the <code>fd-icon-tab-bar--responsive-paddings</code> modifier class.'
+    }
+};
+
+export const paddings = () => `<h4>SM (1rem) paddings</h4>
+<div class="fd-icon-tab-bar fd-icon-tab-bar--sm">
```

Comment:
I thought of using the helper classes (because that's why we added them 😄 ) but the thing is that they are not really consistent and will confuse the user.
This is the specs for IconTabBar:
![Screen Shot 2021-06-18 at 2 58 29 PM](https://user-images.githubusercontent.com/39598672/122605952-2a3f1f00-d046-11eb-9722-47762986b5aa.png)
`sm` and `md` will work because they match the values for paddings:
<img width="585" alt="Screen Shot 2021-06-18 at 3 03 28 PM" src="https://user-images.githubusercontent.com/39598672/122606069-5bb7ea80-d046-11eb-97fc-59b17924fdf9.png">

In these 2 cases we can use `fd-padding-begin-end--sm` and `fd-padding-begin-end--md` but then is the problem with LG, XL and XXL. `fd-padding-begin-end--lg` adds paddings left and right of 3rem but should be 2rem for IconTabBar. Also we don't have paddings for XL and XXL so we will need the additional modifier classes for IconTabBar such as `fd-icon-tab-bar--xxl` and `fd-icon-tab-bar--xl`. For this reason I added `fd-icon-tab-bar--SIZE` for all sizes so that it will be easily handled in the NGX implementation where we will just change the `--SIZE` part of the same class.